tictactoe.Tests/GameTest.cs

```diff
@@ -20,48 +17,74 @@ public GameTest()
         {
             console = new MockConsoleInterface();
             rules = new MockGameRulesInterface();
-            game = new Game(console, rules);
             board = new Board();
             player1 = new Player("X", console);
             player2 = new Player("O", console);
+            game = new Game(console, rules, player1, player2);
         }
 
         [Fact]
         public void GamePlaysThroughTheWholeGameTest()
         {
 
-            console = new MockConsoleInterface();
-            console.setUserInputs(new List<string> { "1", "1", "2", "3", "4", "5", "6", "7", "8", "9" });
-            game = new Game(console, rules);
+            console.setUserInputs(new List<string> { "1", "1", "1", "2", "3", "4", "5", "6", "7", "8", "9" });
+
             game.Menu();
-            Board board = game.GetBoard();
+            Board game_board = game.GetBoard();
```

Comment:
shouldn't vars be camel case?

Reply:
commit 2dc116609a7e9ab9852f1eb31759c51ab3e2d09c